mypyc/lib-rt/CPy.h

```diff
@@ -1042,6 +1042,21 @@ static PyObject *CPyLong_FromFloat(PyObject *o) {
     }
 }
 
+static PyObject *CPyLong_FromStrWithBase(PyObject *o, CPyTagged base) {
+    if (PyLong_Check(o)) {
```

Comment:
Do we need this check? It should always be a str, right?

Reply:
Yes. We should remove it.